modules/apps/commerce/commerce-frontend-js/test/components/AddToCart/AddToCart.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {cleanup, render} from '@testing-library/react';
+import React from 'react';
+
+import AddToCart from '../../../src/main/resources/META-INF/resources/components/add_to_cart/AddToCart';
+
+describe('AddToCart', () => {
+	const INITIAL_PROPS = {
+		channel: {
+			currencyCode: 'USD',
+			id: 12345,
+		},
+		cpInstance: {
+			skuId: 12345,
+		},
+	};
+
+	describe('by display settings', () => {
+		let Component;
+
+		beforeEach(() => {
+			jest.resetAllMocks();
+
+			window.Liferay.Language.get = jest.fn();
+		});
+
+		afterEach(() => {
+			cleanup();
+
+			Component = null;
+		});
+
+		it('renders a Button with the appropriate settings wrapped with its quantity selector elements', () => {
+			const props = {
+				...INITIAL_PROPS,
+				settings: {
+					withQuantity: {
+						allowedQuantities: [],
+						maxQuantity: 10,
+						minQuantity: 1,
+					},
+				},
+			};
+
+			Component = render(<AddToCart {...props} />);
+
+			const WrapperWithQuantityContainer = Component.container.querySelector(
+				'.add-to-cart'
+			);
+			const QuantitySelectorInputElement = Component.container.querySelector(
+				'input'
+			);
+			const AddToCartButtonElement = Component.container.querySelector(
+				'button'
+			);
```

Comment:
I don't think you necessarily need to be resetting and nulling the `Component` variable for each test. This test could look a little more simple

```suggestion
			const {container} = render(<AddToCart {...props} />);

			const WrapperWithQuantityContainer =container.querySelector('.add-to-cart');
			const QuantitySelectorInputElement = container.querySelector('input');
			const AddToCartButtonElement = container.querySelector('button');
```